LDAR_Sim/src/methods/crew.py

```diff
@@ -25,6 +25,9 @@
 from typing import Any, Dict
 
 import numpy as np
+from sklearn.base import OutlierMixin
```

Comment:
What is this import?

Reply:
Oops I have no idea how that made it in - Will remove